README.md

```diff
@@ -1,24 +1,24 @@
-# README
+# Todo app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+ A Todo application built with Rails framework.
+## Instalation
 
-Things you may want to cover:
+ Ruby version:  "3.1.0"
 
-* Ruby version
+ Rails version: 7.0.1"
 
-* System dependencies
+ Install gems: `bundle install`
 
-* Configuration
+ Create database : `bun/rails db:create`
+ ## Start 
 
-* Database creation
+ Run the application: `rails s`
 
-* Database initialization
+ Start the console: `rails c`
 
-* How to run the test suite
+ Start database: `rails db`
```

Comment:
Just a nit, but running `rails` by itself would assume that the user has `rails` installed globally on their machine. Since we're installing rails locally when running `bundle install`, `bin/rails <rails_command>` would work for everyone, while `rails <rails_command>` works for only some.

Reply:
Done